src/test/java/cucumber/setUp/CucumberContext.java

```diff
@@ -0,0 +1,64 @@
+package cucumber.setUp;
+
+import manager.PageFactoryManager;
+import org.apache.log4j.PropertyConfigurator;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.Listeners;
+import pages.HomePage;
+import reporting.TestListener;
+import utils.PropertiesHelper;
+import webDriverFactory.LocalWebDriverCreator;
+import webDriverFactory.RemoteWebDriverCreator;
+import webDriverFactory.WebDriverCreator;
+
+import static java.util.Optional.ofNullable;
+import static utils.PropertiesHelper.getInstance;
+
+@Listeners({TestListener.class})
+public class CucumberContext {
+  public static PropertiesHelper propertiesHelper;
+  public final int FIRST = 1;
+  public PageFactoryManager pageFactoryManager;
+  public WebDriver driver;
+  public HomePage homePage;
+  public int timeout;
+  public String baseUrl;
+  private String hubUrl;
+  private String browser;
+  private boolean localRun;
+
+  private static void configureLog4j() {
+    PropertyConfigurator.configure(propertiesHelper.getProperty("log4jPropertiesPath"));
+  }
+
+  public void setUp() {
+    propertiesHelper = getInstance();
```

Comment:
If I'm not mistaken, you can write like this:
`public static PropertiesHelper propertiesHelper = getInstance();`